src/test/components/TrackContextMenu.test.js

```diff
@@ -515,8 +515,8 @@ describe('timeline/TrackContextMenu', function () {
       expect(getHumanReadableTracks(getState())).toEqual([
         'show [thread GeckoMain default] SELECTED',
         'hide [thread GeckoMain tab]',
-        '  - show [thread DOM Worker]',
-        '  - show [thread Style]',
+        '  - hide [thread DOM Worker]',
+        '  - hide [thread Style]',
```

Comment:
I'm a bit surprised by this change. But maybe that's expected, tell me what you think.

Reply:
So this is a change due to the fact that we changed their `processType` to `tab`. It's not really because of the PR. When we change the `processsType` to `tab` (which is the case for the non-main threads that come from Firefox normally). So now the fixture matches the reality.